services/api/yarn.lock

```diff
@@ -5167,17 +5050,6 @@ jszip@^3.2.2:
     readable-stream "~2.3.6"
     set-immediate-shim "~1.0.1"
 
-juice@^7.0.0:
```

Comment:
so I'm assuming that the reason we can remove juice is because we're not going to inline our styles anymore and we're OK with any loss of mail client support due to this?

Reply:
From everything I can tell we are losing `gmail using imap on mobile` support and everything else is good.
So I would say thats totally a loss worth taking.

Reply:
me too I think it's a major win. we could also play around more with atomic classes in the emails then... this could probably get us pretty far in terms of flexibility of styling without having to create multiple templates

Reply:
yep that might be a good idea, is the current css in the layout.html following the pattern (i suspect not).

Let me make sure the naming is updated to align, what is the convention i am looking for is it BEM ?

Reply:
no bem is different when I say "atomic" I really just mean a few helper classes like `.link` or something... I don't really even know what would be useful yet just saying it could be nice... how this get converted from markdown is a different story so maybe it's a moot point tbh